aosprotocol/aosprotocol.h

```diff
@@ -0,0 +1,34 @@
+/*
+ * Copyright (C) 2023 Renesas Electronics Corporation.
+ * Copyright (C) 2023 EPAM Systems, Inc.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#ifndef AOS_PROTOCOL_H_
+#define AOS_PROTOCOL_H_
+
+#include <stddef.h>
+#include <stdint.h>
+
+#pragma pack(push, 1)
+
+/**
+ * Aos low level protocol header.
+ */
+struct AosProtocolHeader {
+    uint32_t mPort;
+    uint32_t mDataSize;
+    uint8_t mCheckSum[32];
+    uint8_t mData[];
```

Comment:
variable length array is not part of standard C++, it possible to get compiler warning or error if we compile our code with pedantic flag

Reply:
removed